tests/test_app.py

```diff
@@ -188,7 +188,27 @@ def test_packages_empty(testapp):
     assert len(resp.html("a")) == 0
 
 
-def test_health(testapp):
+def test_health_default_endpoint(testapp):
+    resp = testapp.get("/health")
+    assert resp.status_int == 200
+    assert "Ok" in resp
+
+
+def test_health_customized_endpoint(root):
+    from pypiserver import app
+
+    testapp = webtest.TestApp(
+        app(root=root.strpath, health_endpoint="/healthz")
+    )
+    resp = testapp.get("/healthz")
+    assert resp.status_int == 200
+    assert "Ok" in resp
+
+
+def test_health_invalid_customized_endpoint(root):
+    from pypiserver import app
+
+    testapp = webtest.TestApp(app(root=root.strpath, health_endpoint="/simple"))
```

Comment:
Also linked to some comments above, I'd say that here it should raise a `RuntimeError` since this route is already taken :)

Reply:
I think you have a point, Explicit is better than implicit.
https://github.com/pypiserver/pypiserver/pull/442/commits/55fa396999c07be76ed58cfe4b14d860809368a1